src/Engine/Utils.java

```diff
@@ -92,21 +92,25 @@ public static void highlightLocations(GameMap map, ArrayList<XYCoord> locations)
     }
   }
 
-  /** Alias for {@link #findPossibleDestinations(XYCoord, Unit, GameMap, boolean) findPossibleDestinations()} **/
+  /** Alias for {@link #findPossibleDestinations(XYCoord, Unit, GameMap) findPossibleDestinations()} **/
   public static ArrayList<XYCoord> findPossibleDestinations(Unit unit, GameMap gameMap)
   {
     return findPossibleDestinations(new XYCoord(unit.x, unit.y), unit, gameMap);
```

Comment:
Is there any reason not to just make the call straight to `findFloodFillArea` instead of taking a detour through an overload?

Reply:
Both overloads are used and it prevents having to duplicate the logic to find `initialFillPower`.

Reply:
Eh, good enough I guess.